includes/class-wcpdf-install.php

```diff
@@ -216,9 +216,12 @@ protected function upgrade( $installed_version ) {
 		} else {
 			// don't try merging fonts with local when updating pre 2.0
 			$pre_2 = ( $installed_version == 'versionless' || version_compare( $installed_version, '2.0-dev', '<' ) );
-			$merge_with_local = $pre_2 ? false : true;
+			$merge_with_local = !$pre_2;
```

Comment:
Can you explain this? I'm assuming that you're running versions superior to 2.0, so will always be `true` right?

Reply:
Yes, I'm working on versions superior to 2.0, which means that `$merge_with_local` will always be `true`.
Could you please provide a bit more detail about what you'd like me to explain or consider?

Reply:
You made this change for the code to be cleaner?

Reply:
Yes, my intention was to make it cleaner and simpler. Is it ok?

Reply:
Sure, just to clarify! Good work.